pulp_rpm/app/tasks/copy.py

```diff
@@ -0,0 +1,14 @@
+from pulpcore.app.models.repository import RepositoryVersion
+
+
+def copy_content(source_repo, dest_repo):
+    """
+    Copy content from one repo to another.
+
+    Args:
+        source_repo: repository to copy units from
+        dest_repo: repository to copy units to
+    """
+    content_to_copy = RepositoryVersion.latest(source_repo).content
```

Comment:
If we're leaving this in the plugin, I would suggest that we limit it to only copying RPMs and Errata, and not everything in ```.content```

Reply:
@dralley that makes sense to me. Let's also clearly label the feature as only handling those types.